halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/PersistentStore.java

```diff
@@ -40,6 +41,14 @@ public NodeIterator getChildren() {
 
   public void setConnectionInfo(URI uri) {}
 
+  @JsonIgnore
+  public boolean isEnabled() {
```

Comment:
Rather than creating a `isEnabled` method that can cause confusion with other "enabled" settings, I think it's better to create a different interface like `Selectable`, and create a `isSelected` method for this case.

Reply:
The idea was to piggy-back on existing `enabled` flag that exist for many `Node` and always mean the same thing.